client/src/partials/exchange/modal.html

```diff
@@ -10,11 +10,15 @@
 
     <div class="form-group">
       <label>{{ "EXCHANGE.ENTERPRISE_CURRENCY" | translate }}</label>
-      <p class="form-control-static">{{ ModalCtrl.enterpriseCurrency }} : {{ 1 |  currency:ModalCtrl.enterpriseCurrencyId }}</p>
+      <p class="form-control-static">
+        {{ ModalCtrl.enterpriseCurrency }} : {{ 1 |  currency:ModalCtrl.enterpriseCurrencyId }}
```

Comment:
This one is giving the rate proportional to the 1 unit of the enterprise currency (for example, 1 USD or 1 FC).  In this case, I think it is valid.  Not sure about the first one - will investigate further.